projects/bp-strapi/package.json

```diff
@@ -20,6 +20,7 @@
   "dependencies": {
     "@strapi/plugin-graphql": "4.1.9",
     "@strapi/plugin-i18n": "4.1.9",
+    "@strapi/plugin-sentry": "^4.1.12",
```

Comment:
Thats wild. Either update all to 4.1.12 or use the sentry plugin in 4.1.9 please